README.md

```diff
@@ -6,13 +6,13 @@ via the [Checker Framework Language Server](https://github.com/eisopux/checker-f
 
 ## Getting Started
 
-You can either install the extension from the
-[VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
+You can either install the extension from the 
+[VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace) 
 or install a locally-built version of the extension.
 Please refer to the "Developer's Guide" below for instructions on how to build from source.
 
-After installing the extension, when you open or save any `.java` file it will be checked
-by the Checker Framework [Nullness Checker](https://checkerframework.org/manual/#nullness-checker).
+After installing the extension, when you open or save any `.java` file it will be checked 
+by the Checker Framework [Nullness Checker](https://checkerframework.org/manual/#nullness-checker). 
```

Comment:
```suggestion
After installing the extension, when you open or save any `.java` file it will be checked
by the Checker Framework [Nullness Checker](https://checkerframework.org/manual/#nullness-checker).
```